src/azure-cli/azure/cli/command_modules/appservice/static_sites.py

```diff
@@ -191,7 +191,7 @@ def update_staticsite_users(cmd, name, roles, authentication_provider=None, user
 
 def create_staticsites(cmd, resource_group_name, name, location,
                        source, branch, token=None,
-                       app_location='.', api_location='.', app_artifact_location='.github/workflows',
+                       app_location='.', api_location='.', output_location='.github/workflows',
```

Comment:
is this the default?

Reply:
sorry just saw this comment, yes ".github/workflows" is the default